packages/dd-trace/src/appsec/blocking.js

```diff
@@ -5,8 +5,8 @@ const fs = require('fs')
 
 // TODO: move template loading to a proper spot.
 let templateLoaded = false
-let templateHtml = ''
-let templateJson = ''
+let templateHtml = '<html>blocked</html>'
```

Comment:
I think we could require the template file directly here no ?

Reply:
Then it will always be loaded

Reply:
Ah also when a custom config is set you mean ? you're right, maybe we can leave it like this then, up to you

Reply:
Is there some reason for it to _not_ be always loaded? 🤔